src/app/(auth)/login/page.tsx

```diff
@@ -0,0 +1,11 @@
+import LoginForm from "@/components/LoginForm";
+
+const Login: React.FC = () => {
+  return (
+    <>
+      <LoginForm />
```

Comment:
No need to create new component as there is no extra code in this file, the codes from `LoginForm` can be moved here.

Reply:
Done.